src/argus_htmx/templates/htmx/_base_form_modal.html

```diff
@@ -6,7 +6,10 @@
 <dialog id="{{ dialog_id }}" class="modal">
     <div class="modal-box card card-compact shadow-xl">
         <div class="divider divider-start"><h3 class="card-title">{{ header }}</h3></div>
-        <form id="{{ dialog_id }}-form" class="card-body" method="post" action="{{ endpoint }}">
+        <form id="{{ dialog_id }}-form" class="card-body"
+              hx-post="{% url endpoint|default:'htmx:incident-list' %}"
+              hx-include="[name='selected_incidents']"
+        >
```

Comment:
This file is called base_form_modal. Does it make sense to add the hx-include in here? Perhaps rename to base_incident_update_modal? Or, if this modal is used elsewhere for not updating incidents, make it something else?

Reply:
There's also #127

Reply:
We have some other modals that we use to display some data:
https://github.com/GEANT/geant-argus/blob/develop/src/geant_argus/geant_argus/templates/htmx/incidents/_incident_quick_glance_modal.html
https://github.com/GEANT/geant-argus/blob/develop/src/geant_argus/geant_argus/templates/htmx/incident_details/event_properties.html
But no need to make a generic component out of it

We may want to look at extending base_form_modal for our own custom incident bulk edit modals. 

But the main point I wanted to make here is that base_form_modal.html is used in other places in argus_htmx and you don't necessarily want to hx-include selected_incidents there

Reply:
sure, but I think it should read
```
            {% else %}
          method="post"
          action="{% url endpoint|default:'htmx:incident-list' %}"
            {% endif %}
```